client/src/main/scala/drt/client/components/styles/DrtTheme.scala

```diff
@@ -113,4 +113,30 @@ object DrtTheme extends StyleSheet.Inline with ScalaCssReactImplicits {
       ),
     )
   )
+
+  def accessibilityTheme: Theme = {
```

Comment:
I think Jackson created a theme in the react lib so we can add this to it if these are overrides and then just use the react component theme